openslides_backend/action/actions/user/set_present.py

```diff
@@ -89,3 +93,23 @@ def check_permissions(self, instance: Dict[str, Any]) -> None:
         ):
             return
         raise PermissionDenied("You are not allowed to set present.")
+
+    def get_history_information(self) -> Optional[List[str]]:
+        # use present_values to get present_str
+        present_str = "(not) "
+        if len(self.present_values) == 1 and list(self.present_values)[0]:
```

Comment:
```suggestion
        if len(self.present_values) == 1 and self.present_values.pop():
```
same for all other instances where you cast single-membered sets to lists

Reply:
Be careful, pop removes one value from self.present_values. I don't want that.

Reply:
Okay, I updated the code to use `pop`